int64 InstanceSize
string LabelingType
---
int64 Size
int64 Changed
int64 InstanceClass
int64[] GTClasses